src/server/pfs/cmds/cmds.go

```diff
@@ -66,7 +66,7 @@ Repos are created with create-repo.`,
 		Short: "Create a new repo.",
 		Long:  "Create a new repo.",
 		Run: cmd.RunFixedArgs(1, func(args []string) error {
-			client, err := client.NewFromAddress(address)
+			client, err := client.NewUserClientFromAddress(address)
```

Comment:
So doesn't this mean that this method reports metrics and there's no way for the user to turn them off? And isn't this true for every other method as well except `version` and `deploy`?